woocommerce-gateway-ebanx.php

```diff
@@ -403,8 +406,8 @@ public function my_account_template() {
 
 			$cards = array_filter(
 				(array) get_user_meta( get_current_user_id(), '_ebanx_credit_card_token', true ), function ( $card ) {
-					return ! empty( $card->brand ) && ! empty( $card->token ) && ! empty( $card->masked_number ); // TODO: Implement token due date.
-				}
+				return ! empty( $card->brand ) && ! empty( $card->token ) && ! empty( $card->masked_number ); // TODO: Implement token due date.
```

Comment:
```suggestion
				return ! empty( $card->brand ) && ! empty( $card->token ) && ! empty( $card->masked_number );
```

Reply:
Please, do not send code comments to the production environment

Reply:
This comes way before then I started coding. I don't know who and why this comment was added. Anyway, this is out of log scope.

Reply:
I've removed the comment